packages/lib-classifier/src/plugins/tasks/TextTask/components/TextTask/TextTask.js

```diff
@@ -70,7 +71,7 @@ function TextTask (props) {
           autoFocus={autoFocus}
           disabled={disabled}
           id={`${task.taskKey}-${task.type}`}
-          defaultValue={value}
+          value={value}
```

Comment:
I wonder if this is uncontrolled under the hood, because it still works even though the code never updates `value` on change, as far as I can tell.

Reply:
It's controlled. I checked their source code when debugging this.

Reply:
For ref: https://github.com/grommet/grommet/blob/master/src/js/components/TextArea/TextArea.js

All of the form components are using hooks and context now for setting the value.

Reply:
Interesting. So the value that's passed in here is used as the default value for `FormContext` in the `TextArea` code, then the input updates `FormContext` via `setValue`. I was trying to figure out how this was working since our code doesn't directly control `value`.